src/main/java/org/zeromq/ZMQException.java

```diff
@@ -4,17 +4,8 @@
 
 public class ZMQException extends RuntimeException
 {
-    public static class IOException extends RuntimeException
-    {
-        private static final long serialVersionUID = 8440355423370109164L;
-
-        public IOException(java.io.IOException cause)
-        {
-            super(cause);
-        }
-    }
 
-    private static final long serialVersionUID = 5957233088499712341L;
+    private static final long serialVersionUID = -978820750094924644L;
```

Comment:
I compared jzmq and jeromq side by side and moved over a number of blocks that were different, including this value. To be fair, it should not make any difference, and I could as well have have left the value unchanged as i don't think anyone will be serializing this exception